includes/format.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#ifdef __has_include
+#if __has_include(<format>)
+#include <format>
+#endif
+#endif
+
+#ifdef __cpp_lib_format
+#if __cpp_lib_format >= 201907L
+#define CR_HAS_CPP20_FORMAT
+#endif
+#endif
+
+#if !defined(CR_HAS_CPP20_FORMAT)
+#include <fmt/core.h>
+// Inject fmt:: stuff to std::, since fmtlib inplemented C++20 std::format
```

Comment:
Please don't, it doesn't guarantee to be compatible to every STL.
If you have to inject something, only alias those, don't inject the entire namespace.